src/Honeycomb.OpenTelemetry/HoneycombOptions.cs

```diff
@@ -13,18 +13,17 @@ namespace Honeycomb.OpenTelemetry
     public class HoneycombOptions
     {
         private const string OtlpVersion = "0.16.0";
+        private const string OtelExporterOtlpProtcolHttp = "http/protobuf";
+        private const string OtelExporterOtlpProtcolGrpc = "grpc";

```

Comment:
```suggestion
        private const string OtelExporterOtlpProtocolGrpc = "grpc";
```
suggest rename symbol (I don't know why my eyes notice these things)